tiledb/dataframe_.py

```diff
@@ -569,6 +573,20 @@ def from_pandas(uri, dataframe, **kwargs):
             }
         )
 
+    # handle coords/offsets filters
+    if coords_filters is True:
+        coords_filters = None
+    elif coords_filters is None:
+        # empty coords filters support
+        coords_filters = tiledb.FilterList()
+
+    if offsets_filters is True:
+        offsets_filters = None
+    elif offsets_filters is None:
+        # empty offsets filters support
+        offsets_filters = tiledb.FilterList()
+
```

Comment:
1. Can `coords_filters` and `offsets_filters` be a list? If yes shouldn't they be wrapped in a ` tiledb.FilterList`?
2. Unlike `attr_filters` and `dim_filters` no ValueError is raised here for invalid filters.

Reply:
Updated, thanks.